org.kde.ktorrent.json

```diff
@@ -0,0 +1,98 @@
+{
+    "id": "org.kde.ktorrent",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "ktorrent",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=host",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=${FLATPAK_DEST}",
+                "./b2 -j ${FLATPAK_BUILDER_N_JOBS} headers",
+                "mkdir -p ${FLATPAK_DEST}/include/boost",
+                "cp -r boost/* ${FLATPAK_DEST}/include/boost"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2",
+                    "sha256": "83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1"
+                }
+            ]
+        },
+        {
+            "name": "qca",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DQT4_BUILD=OFF",
```

Comment:
you don't need this, there's no qt4 build anymore, you don't need to set the QCA_SUFFIx either

Reply:
Fixed, thanks!